maps/triumph/levels/deck4.dmm

```diff
@@ -6334,9 +6334,13 @@
 /area/security/armoury)
 "eGK" = (
 /obj/structure/table/woodentable,
-/obj/item/radio/off,
-/obj/item/megaphone,
-/obj/item/storage/box/glasses/rocks,
+/obj/item/radio/off{
+	pixel_x = -7;
+	pixel_y = 2
+	},
+/obj/item/megaphone{
```

Comment:
you should already get a megaphone in your locker

Reply:
moved a spare one that was already there